Mathlib/Topology/Algebra/Module/LinearMap.lean

```diff
@@ -60,6 +60,10 @@ abbrev ContinuousLinearMapClass (F : Type*) (R : outParam Type*) [Semiring R]
     [TopologicalSpace M₂] [AddCommMonoid M₂] [Module R M] [Module R M₂] [FunLike F M M₂] :=
   ContinuousSemilinearMapClass F (RingHom.id R) M M₂
 
+/-- The topological dual of a topological space `M`. -/
+abbrev TopologicalSpace.Dual (R : Type*) [Semiring R] [TopologicalSpace R]
+  (M : Type*) [TopologicalSpace M] [AddCommMonoid M] [Module R M] : Type _ := M →L[R] R
```

Comment:
I don't love the idea of this living in the `TopologicalSpace` namespace, as it requires more than just a topological space. I think this naming might be more common.
```suggestion
/-- The *strong dual* of a topological vector space space `M`. This is the space of 
continuous linear functionals and is equipped with the topology of uniform convergence
on bounded subsets. -/
abbrev StrongDual (R : Type*) [Semiring R] [TopologicalSpace R]
  (M : Type*) [TopologicalSpace M] [AddCommMonoid M] [Module R M] : Type _ := M →L[R] R
```